source/1_engine/rex_engine/src/filesystem/vfs.cpp

```diff
@@ -1,5 +1,6 @@
 #include "rex_engine/filesystem/vfs.h"
```

Comment:
I think the first auto tests should be written for this class ...
Even just to test the API

Reply:
yes, I agree, but outside of this PR.
this PR is just implementing autotests and adds 1 as an example how the API should be used

Reply:
Create an issue for this and this conversation is also resolved